recipes/android-ndk/all/conanfile.py

```diff
@@ -0,0 +1,207 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class AndroidNDKInstallerConan(ConanFile):
+    name = "android-ndk"
+    description = "The Android NDK is a toolset that lets you implement parts of your app in " \
+                  "native code, using languages such as C and C++"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://developer.android.com/ndk/"
+    topics = ("NDK", "android", "toolchain", "compiler")
+    license = "Apache-2.0"
+    short_paths = True
+    no_copy_source = True
+    exports_sources = ["cmake-wrapper.cmd", "cmake-wrapper"]
+
+    settings = {"os": ["Windows", "Linux", "Macos"],
+                "arch": ["x86_64"]}
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        if self.settings.arch != 'x86_64':
+            raise ConanInvalidConfiguration("No binaries available for other than 'x86_64' architectures")
+
+    def source(self):
+        tarballs = self.conan_data["sources"][self.version]
+        tools.get(**tarballs[str(self.settings.os)])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
```

Comment:
I think the downloading of the binaries should move to `build`,
because the following will fail:
```
conan create android-ndk@r21d -s Windows
conan create android-ndk@r21d -s Linux -ks
```

Reply:
I do not understand
If you want to download this for a operating system you are not on, this package will not make any sense

Reply:
This would make it possible for a devops guy to create android-ndk packages on his Linux machine for all platforms.
This is possible now too, but there is a possibility that, by executing the commands in my previous comment, a wrong package is generated.

Reply:
This is actually a good point I never thought about because I have all 3 OSes just available,
but I sometimes found it annoying to change the computer 
So I will investigate in this, thanks!